### Mapper ################################################

grid_resolution: 0.2
range_threshold: 10.0 #5.9999999999999999
map_update_rate: 5
publish_pose_graph: false
max_covariance: 0.01
transform_publish_period: 0.05
min_map_size: 3

### Localizer #############################################

min_particles: 2500
max_particles: 10000

### Karto #################################################

# For a full list of available parameters and their
# corresponding default values, see OpenKarto/OpenMapper.h

MinimumTravelDistance: 1.0
MinimumTravelHeading: 1.57
UseScanBarycenter: false
UseScanMatching: false
#ScanBufferSize: 2
#DistanceVariancePenalty: 0.01
#AngleVariancePenalty: 0.01
#LinkScanMaximumDistance: 1.0
#LoopSearchMaximumDistance: 100.0
#LinkMatchMinimumResponseFine: 1.0

#LoopMatchMinimumChainSize: 10000000
###########################################################

